components/mdc/Drawer/Drawer.svelte

```diff
@@ -2,9 +2,9 @@
 <script>
 import { isAboveTablet as isDesktop } from '../breakpoints'
 import { MDCDrawer } from '@material/drawer'
+import { Button, TopAppBar } from '../index'
```

Comment:
I'm not sure if we were trying to organize the imports in `ui-components` (we decided to do that with the riskman), but maybe put this above `MDCDrawer` near the other "local" import.